tasks/install_shell.sh

```diff
@@ -67,13 +67,19 @@ assert_unmodified_apt_config() {
 }
 
 # Check whether perl and LWP::Simple module are installed
-exists_perl() {
-  if perl -e 'use LWP::Simple;' >/dev/null 2>&1
-  then
+exists_perl_lwp() {
+  if perl -e 'use LWP::Simple;' >/dev/null 2>&1 ; then
+    return 0
+  fi
+  return 1
```

Comment:
These can be simplified:

```suggestion
  perl -e 'use LWP::Simple;' >/dev/null 2>&1
```

Reply:
I generally kept the same format as the existing perl function to make it easy to review and approve.

Reply:
The entire script is littered with this type of syntax. Happy to open a separate PR to "write" all the wrongs.